core/src/main/scala/org/http4s/blaze/channel/nio1/FixedSelectorPool.scala

```diff
@@ -0,0 +1,27 @@
+package org.http4s.blaze.channel.nio1
+
+import java.nio.channels.Selector
+import java.util.concurrent.atomic.AtomicInteger
+
+/** Provides a fixed size pool of [[SelectorLoop]]s, distributing work in a round robin fashion */
+class FixedSelectorPool(poolSize: Int, bufferSize: Int) extends SelectorLoopPool {
+
+  private val loops = 0
+    .until(poolSize)
+    .map { id =>
+      val l = new SelectorLoop(s"blaze-nio-fixed-selector-pool-$id", Selector.open(), bufferSize)
+      l.setDaemon(true)
```

Comment:
Probably, but I'd like to save the behavior change for the `ThreadFactory` issue since this is about cleanup and not behavior change. Fft comment on that issue so I don't forget to address it.

Reply:
Btw, this is issue #183.